package.json

```diff
@@ -44,12 +44,12 @@
   "homepage": "https://github.com/angular/angular-cli",
   "dependencies": {
     "@angular-cli/ast-tools": "^1.0.0",
-    "@angular/common": "^2.0.0",
-    "@angular/compiler": "^2.0.0",
+    "@angular/common": "~2.1.0",
+    "@angular/compiler": "~2.1.0",
     "@angular/compiler-cli": "^0.6.0",
```

Comment:
Per this [comment](https://github.com/angular/angular-cli/pull/2704#issuecomment-254058507), could you also update the version of `@angular/compiler-cli` [npm](https://www.npmjs.com/package/@angular/compiler-cli) has it's latest version as `2.1.0`